docs/log_collection.md

```diff
@@ -0,0 +1,184 @@
+# Android Log Collection
+
+Send logs to Datadog from your Android applications with [Datadog's `dd-sdk-android` client-side Kotlin logging library][1] and leverage the following features:
```

Comment:
I'm not sure we should specifically say `Kotlin` in this sentence. It might give the idea that the library might only work with Kotlin code, when in fact it is also compatible with Java code. I'd remove Kotlin entirely and simply add a line saying that the library can be used from both Kotlin and Java code (eg as stated in the list below, line 7)